anndata/_io/write.py

```diff
@@ -103,7 +103,11 @@ def write_loom(filename: PathLike, adata: AnnData, write_obsm_varm: bool = False
     for key in adata.layers.keys():
         layers[key] = adata.layers[key].T
 
-    from loompy import create
+    from numba.core.errors import NumbaDeprecationWarning
+
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore", NumbaDeprecationWarning)
+        from loompy import create
```

Comment:
```suggestion
    from loompy import create
```

I don't know that it's our place to hide this warning. I wouldn't want people to think our implementation was in a better state than loompy's since it doesn't warn.